cws-test/src/test/resources/hostname.txt

```diff
@@ -1 +1 @@
-localhost
+localhost
```

Comment:
I think this should maybe be included in the .gitignore.  Delete this file from the repo.
Add it as:
hostname.txt

Reply:
Yes. I also added:
jacoco-reports/
target/

to .gitignore